pipestat/helpers.py

```diff
@@ -229,3 +232,26 @@ def get_all_result_files(results_file_path: str) -> List:
     files = glob.glob(results_file_path + "**/*.yaml")
 
     return files
+
+
+def zip_report(report_dir_name: str):
+    """
+
+    Walks through files and attempts to zip them into a Zip object using default compression.
+    Gracefully fails and informs user if compression library is not available.
+
+    :param report_dir_name: directory name of report directory
+    :return: None
+    """
+
+    zip_file_name = report_dir_name + "_report_portable"
+
+    try:
+        make_archive(zip_file_name, "zip", report_dir_name)
+    except RuntimeError as e:
+        _LOGGER.warning("Report zip file not created! \n {e}")
+
+    if os.path.exists(zip_file_name + ".zip"):
+        _LOGGER.info(f"Report zip file successfully created: {zip_file_name}")
+    else:
+        _LOGGER.warning("Report zip file not created.")
```

Comment:
No. I want to create the directory and then try to zip it. If the user's os does not have the proper library, I want zipping to fail gracefully (not throw an exception) so that they can take the portable folder and just zip it themselves.